project_app/components/Accordion.js

```diff
@@ -0,0 +1,53 @@
+import Accordion from '@mui/material/Accordion';
+import AccordionSummary from '@mui/material/AccordionSummary';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import styled from 'styled-components';
+
+function AccordionLayout() {
+  return (
+    <div>
+      <Accordion
+        style={{
+          backgroundColor: 'rgb(53, 53, 53)',
+          width: '50%',
+          marginLeft: '300px',
+          border: '2px solid white'
+        }}
+      >
+        <AccordionSummary
+          expandIcon={<ExpandMoreIcon style={{ color: 'white' }} />}
+        >
+          <Typography>FAQ 1</Typography>
+        </AccordionSummary>
+      </Accordion>
```

Comment:
this is repeating. there's a pattern, this in itself should be a reusable component

Reply:
Oh got it - working on it